models/zendesk__ticket_metrics.sql

```diff
@@ -89,16 +89,16 @@ select
   ticket_comments.count_internal_comments as total_agent_replies,
   
   case when ticket_enriched.is_requester_active = true and ticket_enriched.requester_last_login_at is not null
-    then (({{ dbt_utils.datediff("ticket_enriched.requester_last_login_at", dbt_utils.current_timestamp(), 'second') }} /60))
+    then round(({{ dbt_utils.datediff("ticket_enriched.requester_last_login_at", dbt_utils.current_timestamp(), 'second') }} /60),2)
```

Comment:
```suggestion
    then({{ dbt_utils.datediff("ticket_enriched.requester_last_login_at", dbt_utils.current_timestamp(), 'second') }} /60)
```
A previous Postgres bug was caused by these rounding functions. We have since removed them from the package.